recipes/aif360/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "aif360" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c4b44742984c741e6dab1770afc65480b1fcd8e04d33549c346b03674b6b8dd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5, <3.8
+  run:
+    - matplotlib-base
+    - numpy >=1.16
+    - pandas >=0.24.0
+    - python >=3.5, <3.8
+    - scikit-learn >=0.21
+    - scipy >=1.2.0
+
+test:
+  imports:
+    - aif360
+    - aif360.algorithms
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/IBM/AIF360
+  summary:  A comprehensive set of fairness metrics for datasets and machine learning models, explanations for these metrics, and algorithms to mitigate bias in datasets and models. 
+  description: |
+    The AI Fairness 360 toolkit is an extensible open-source library containing techniques developed by the research 
+    community to help detect and mitigate bias in machine learning models throughout the AI application lifecycle. 
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
Add comment in the line above `license_file` that looks something like:

```
...
# License file manually packaged. See: https://github.com/Trusted-AI/AIF360/pull/206
license_file: LICENSE
```

I've opened https://github.com/Trusted-AI/AIF360/pull/206 upstream and the comment will be a reminder to check to see if the license file is included in future releases so that the one included besides the `meta.yaml` can be removed.